src/contexts/tokens.tsx

```diff
@@ -31,8 +32,9 @@ const NEON_TOKEN_MODEL = {
 
 export function TokensProvider({ children = undefined }) {
   const { get } = useHttp()
-  const { chainId } = useNetworkType()
```

Comment:
@Tkachez, without `chainId` method `mergeTokenList` will fail for all wallets except Metamask. Check it please.